### DISCUSSION TITLE
CD RIPPING

### DISCUSSION THREAD

**Discussion by JerryG51** [smplayer-dev/smplayer#230] [Q&A] [answered]
Hi this is my first time starting discussion. I am a seventy years old very young man that loves walking and filming in the Italian mountains during my holiday.  I use SMPlayer as my favoutite player of my films and I have a question : pls someone could tell me if is it possible ripping a CD or CD single tracks with SMPlayer ? If so what is the correct procedure ? Thanks you in advance.
JerryG51

**Comment by smplayer-dev** [comment, marked as answer]
You can play CDs but there's no option to rip CDs.

**Comment by JerryG51** [comment]
Thank you.  Pls could you close my question ?
